user_modules/dummy_connecttest.lua

```diff
@@ -371,37 +371,48 @@ end
 --! @example: self:initHMI_onReady(local_hmi_table) ]]
 function module:initHMI_onReady(hmi_table)
   local exp_waiter = commonFunctions:createMultipleExpectationsWaiter(module, "HMI on ready")
-
+  local delayValue = 0
   local function ExpectRequest(name, hmi_table_element)
-    if hmi_table_element.occurrence == 0 then
-      EXPECT_HMICALL(name, hmi_table_element.params)
-      :Times(0)
-      commonTestCases:DelayedExp(3000)
-      return
-    end
     local event = events.Event()
     event.level = 2
     event.matches = function(self, data)
       return data.method == name
     end
+
     local exp = EXPECT_HMIEVENT(event, name)
-    :Times(hmi_table_element.mandatory and 1 or AnyNumber())
+    :Times(hmi_table_element.occurrence or hmi_table_element.mandatory and 1 or AnyNumber())
```

Comment:
@aderiabin, please specify expression explicitly.

Reply:
@Itileda Specified in 01408c5